WORKSPACE

```diff
@@ -34,3 +34,19 @@ pip_install(
 
 load("@//tensorflow:workspace.bzl", "workspace")
 workspace()
+
+http_archive(
+  name = "pybind11_bazel",
+  strip_prefix = "pybind11_bazel-72cbbf1fbc830e487e3012862b7b720001b70672",
+  urls = ["https://github.com/pybind/pybind11_bazel/archive/72cbbf1fbc830e487e3012862b7b720001b70672.zip"],
+)
+
+http_archive(
+  name = "pybind11",
+  build_file = "@pybind11_bazel//:pybind11.BUILD",
+  strip_prefix = "pybind11-master",
+  urls = ["https://github.com/pybind/pybind11/archive/refs/heads/master.tar.gz"],
+)
+
+load("@pybind11_bazel//:python_configure.bzl", "python_configure")
```

Comment:
this is nitpicking a bit. I recall bazel's recommendation is put all load statement right at the top. Feel free to address this later